docs/_docs/Instructors/01-Teaching-process.md

```diff
@@ -0,0 +1,173 @@
+the timelines represented here follow that logic and provide rough estimates of the months
+where each part of the process happens. This should serve as a base for volunteers to
+know when to get involved in different parts of the process.
+
+## Batch kick-start
+
+In between batches we take student feedback and together with the rest of the 
+organization debate potential changes to the structure and contents of the academy.
+There are two main focus:
+
+- The taught contents are revised and potentially updated - the final responsibility lies
+on the curriculum AOR
+- The schedule and structure of teaching are discussed and adjusted with student feedback -
+the main responsibility lies on teaching AOR
+
+After changes are discussed and closed, which typically happens during or around a 
+general summit - prepared through github issues and shared in slack, the process follows 
+with documenting and allocating volunteers for the next batch:
+
+<img src="../../images/batch-start-timeline.png"/>
+
+It's at this point that volunteers should be proactive and share where they would like
+to work. All allocation is done through our [github's main repo](https://github.com/LDSSA/wiki/)
+and is the responsibility of the teaching lead and the specialization leads (within a specialization)
+to coordinate.
+
+## Leaning units development
+
+Most of the time, the curriculum won't change drastically in between batches. Actually,
+in the majority of cases there are only slight adjustments or no adjustments at all regarding
+the desired contents of the units. 
+
+With this in mind, our process is designed to build
+on top of the previous batch and improve the materials further. This allows us to take 
+units that are already high quality and push them even further.
+
+In general, the focus when improving an existing unit should be:
+* Review the existing materials: go through the learning and example notebooks 
+and check for potential improvements - maybe something isn't fully clear or would 
+benefit from an extra example; this will also give you a chance of getting familiar
+with the unit
+* Revamp the exercise notebook: change the exercises within reasonable so they are 
+not the same as the previous batches: it is up to you if you want to design a completely
+new notebook, including the dataset and narrative, or if you want to work with the unit
+already has set up.
+
+This process can happen anytime from the moment the allocation is set until the unit is 
+released but in its limit:
+* the instructor should start working at least two months before the unit is released
+* the instructor should deliver the unit to QA at least a month before the unit is to be released to students
+* QA should deliver feedback as fast as possible, but all feedback should be delivered at least a week before 
+unit is to be released to students
+* the instructor should be available to support students via slack asynchronously during the
```

Comment:
I would add 
* The instructor should be available for at least one hour during the time of the specialization for office hours with the students

Reply:
Added